OfficeApp3Web/Content/AppV2.css

```diff
@@ -540,6 +540,18 @@ li.listItem:hover{
     padding: 5px;
 }
 
+#sampleSelector:focus {
+    border-color: black;
+    border-style: dotted;
+    border-width: 1.5px;
+}
+
+#groupSelector:focus {
+    border-color: black;
+    border-style: dotted;
+    border-width: 1.5px;
+}
+
 @media screen and (-ms-high-contrast:none){

```

Comment:
Same thing for line 569 and 574 below. Please test on different browser and desktop also to make sure all working.